app/frontend/src/components/minesattestations/admin/inspection/StatusTable.vue

```diff
@@ -37,18 +27,17 @@ import minesAttestationsService from '@/services/minesAttestations/minesAttestat
 export default {
   name: 'StatusTable',
   props:{
-    ipcPlanId: {
+    submissionId: {
       required: true,
       type: String
     }
   },
   data() {
     return {
       headers: [
-        { text: 'Status', value: 'status' },
+        { text: 'Status', value: 'code' },
         { text: 'Date', align: 'start', value: 'createdAt' },
-        { text: 'Grade', align: 'start', value: 'grade' },
-        { text: 'Assigned Inspector', value: 'inspectorName' },
+        { text: 'Assignee', value: 'inspectorName' },
         { text: 'Inspection Date', value: 'inspectionDate' }
```

Comment:
Should talk to Matt.  Pretty sure he wants to move away from the term Inspection and Inspection Status.  Make it a more general thing and that the status is the admin workflow.  It gets kind of confusing when part of the status is to deal with how it is moving through the forms app and other statuses are how it is moving through inspection states (basically an external workflow).